### PR TITLE
Equalizer

### PR DESCRIPTION
**Pull request by unidrelyn**
## Summary

it creates files of 'css' equalizer and styles, they are compiled in application file

## Screenshots

image

## TrelloCard

https://trello.com/c/EQZiWAeo/19-css-resets

### CODE REVIEW

File: Maquetacion/index.html [hunk 1]
```diff
@@ -9,5 +9,6 @@
   </head>
   <body>
     <h1 class="title">Hello World</h1> 
+    <p> kajsfchskjadfsjadfb</p>
```
Comment (jmtov) [inline Maquetacion/index.html]: Please remove this line. 😅

File: Maquetacion/scss/styles.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.title {
+    color: red;
+}
+
```
Comment (jmtov) [inline Maquetacion/scss/styles.scss]: Remember that the last line should be empty. There seems to be some spaces here.

**Review by jmtov — CHANGES_REQUESTED** [review]
Nice job here too! Please check out my comments and we're good to go. ✌️

File: Maquetacion/scss/styles.scss [hunk 3]
```diff
@@ -1,4 +1,3 @@
 .title {
     color: red;
 }
-
```
Comment (jmtov) [inline Maquetacion/scss/styles.scss]: Did you remove the last line here?

**Review by jmtov — CHANGES_REQUESTED** [review]

File: Maquetacion/index.html (hunk 1 above)
Comment (unidrelyn) [reply Maquetacion/index.html]: 👍

File: Maquetacion/scss/styles.scss (hunk 2 above)
Comment (unidrelyn) [reply Maquetacion/scss/styles.scss]: 👍

**Review by unidrelyn — COMMENTED** [review]
I do not remember, but it's already corrected

**Review by jmtov — APPROVED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**